<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificaAcquisto</class>
 <widget class="QWidget" name="ModificaAcquisto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifica Acquisto</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scegli i parametri che vuoi modificare:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data di acquisto: </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="line_data">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_data">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Importo totale:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_importo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_importo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
